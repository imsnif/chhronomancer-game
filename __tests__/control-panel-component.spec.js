import { Text } from 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

jest.mock('mobx-react/native', () => require('mobx-react/custom'))

test('Control Panel => renders correctly', () => {
  const ControlPanel = require('../components/control-panel').default
  const tree = renderer.create(
    <ControlPanel>
      <Text>foo</Text>
      <Text>bar</Text>
    </ControlPanel>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
