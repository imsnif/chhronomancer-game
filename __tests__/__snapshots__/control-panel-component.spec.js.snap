// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Control Panel => renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": "20%",
        "resizeMode": "stretch",
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 2,
          "marginTop": 5,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        foo
      </Text>
    </View>
    <View
      style={
        Object {
          "marginLeft": 4,
        }
      }
    />
    <View
      style={
        Object {
          "flex": 1,
          "marginTop": 5,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        bar
      </Text>
    </View>
  </View>
</View>
`;
